Automattic/wp-calypso
# 100-year-plan: Embed Calendly Widget Popup

## Proposed Changes

Applied feedback from the previous two PRs that will be closed since this PR is replacement:
- https://github.com/Automattic/wp-calypso/pull/95195
- https://github.com/Automattic/wp-calypso/pull/95257

* Updated and integrated the Calendly Popup widget
* Adjusted styles to match the design
* Matched "Thank you" step design showing Earth video

## Testing Instructions

- Go to `/setup/hundred-year-plan/diy-or-difm?flags=100year%2Fvip`
- Click on "Schedule your fee call"
- Check the embedded Calendly Popup
- Schedule an event
- Check if you ended up in Thank you step

<img width="1840" alt="Screenshot 2024-10-10 at 23 45 45" src="https://github.com/user-attachments/assets/dd2f6969-087d-48e4-8655-93f16da99acc">
<img width="1840" alt="Screenshot 2024-10-11 at 14 11 07" src="https://github.com/user-attachments/assets/f4bfb137-d375-49c6-b5f5-7b5efe8a31af">




## Pre-merge Checklist
